.tool-versions

```diff
@@ -0,0 +1 @@
+java adoptopenjdk-16.0.2+7
```

Comment:
This came from Andy's notes on how he set up his macbook: https://mojdt.slack.com/archives/C025S4PG5FT/p1627481715034000?thread_ts=1627383933.012900&cid=C025S4PG5FT